pepper-apis/src/main/resources/db-changes/seed/spanish-and-creole.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20200716-add-creole-language-code">
+        <insert tableName="language_code">
+            <column name="iso_language_code" value="ht"/>
+        </insert>
+    </changeSet>
+
+    <changeSet author="yufeng" id="20200716-spanish-creole-activity-type">
+        <update tableName="i18n_activity_type_name_trans">
+            <column name="translation_text" value="Formularios"/>
```

Comment:
@charlottedye These are the platform strings we're going to use for Spanish `es`. Will these work for Prion?

Reply:
I feel sure the translations will work for Prion since I imagine it's even the same translation company.